rotkehlchen/globaldb/schema.py

```diff
@@ -162,6 +162,13 @@
 );
 """
 
+DB_CREATE_ADDRESS_BOOK = """
+CREATE TABLE IF NOT EXISTS address_book (
+    identifier TEXT NOT NULL PRIMARY KEY,
+    name TEXT /* can be null */
+);
+"""
```

Comment:
The idea was that it's allowed to put here any identifier and get data for it in the frontend when needed. I thought that we could potentially use it in more places and not only addresses. But maybe it will make sense to make in an address for now

Reply:
Yes, I tough we had a table in the globaldb for this. Maybe we can create one with the chains we support? What do you think @LefterisJP I believe that would be interesting to have this filter in the future